prod.cljs.edn

```diff
@@ -1,4 +1,6 @@
+^{:auto-bundle :webpack}
 {:main com.yetanalytics.lrs-admin-ui.core
+ :target :bundle
  :externs ["externs/react.ext.js"
            "externs/react-dom.ext.js"]
  :language-out :es6
```

Comment:
When using `^{:auto-bundle :webpack}` you should be able to safely omit `:target :bundle` and `:language-out :es6`, per testing on my old branch

Reply:
unless something upstream has changed in figwheel

Reply:
nope you are correct. i started manually configuring then forgot to take this out. doesnt hurt but is extraneous

Reply:
done

Reply:
I was also able to remove `:language-out` since I'm pretty sure that also provides webpack settings and I don't know if it matches the default

Reply:
but if it is working I'm inclined to leave it.. though I think it does mean the browser has to be es6-aware?

Reply:
or maybe that's the language out of Closure that goes to webpack. not sure